creature/badooga; Better Greatwyrms.json

```diff
@@ -0,0 +1,2117 @@
+					]
+				},
+				{
+					"name": "Tail",
+					"entries": [
+						"{@atk mw} {@hit 18} to hit, reach 20 ft., one target. {@h}19 ({@damage 2d8 + 10}) bludgeoning damage. If the target is a creature, it must succeed on a {@dc 26} Strength saving throw or be knocked {@condition prone}."
+					]
+				},
+				{
+					"name": "Frightful Presence",
+					"entries": [
+						"Each creature of the greatwyrm's choice that is within 1 mile of the dragon and aware of it must succeed on a {@dc 19} Wisdom saving throw or become {@condition frightened} for 1 minute. A creature can repeat the saving throw at the end of each of its turns, ending the effect on itself on a success. If a creature's saving throw is successful or the effect ends for it, the creature is immune to the greatwyrm's Frightful Presence for the next 24 hours."
+					]
+				},
+				{
+					"name": "Frost Breath {@recharge 5}",
+					"entries": [
+						"The greatwyrm exhales an icy blast in a 300-foot cone. Each creature and object in that area takes 88 ({@damage 16d10}) cold damage ({@dc 25} Constitution save for half). Damaged surfaces in this area then become {@hazard slippery ice} (DMG 110) until cleared."
+					]
+				}
+			],
+			"reaction": [
+				{
+					"name": "Vicious Reprisal",
+					"entries": [
+						"In response to taking damage, the greatwyrm makes a bite attack against a random creature within 5 feet of it. If no creature is within reach, the greatwyrm can attack an object, or it can move up to half its speed toward an enemy it can see, without provoking opportunity attacks."
+					]
+				}
+			],
+			"legendary": [
+				{
+					"name": "Tail Attack",
+					"entries": [
+						"The greatwyrm makes two tail attacks, one of which must be against an object."
+					]
+				},
+				{
+					"name": "Savage (Costs 2 Actions)",
+					"entries": [
+						"The greatwyrm makes a claw attack against each creature and object within 10 feet of it."
+					]
+				},
+				{
+					"name": "Wing Attack (Costs 2 Actions)",
+					"entries": [
+						"The greatwyrm beats its wings. Each creature within 10 feet of the greatwyrm must succeed on a {@dc 26} Dexterity saving throw or take 17 ({@damage 2d6 + 10}) bludgeoning damage and be knocked {@condition prone}. The greatwyrm can then fly up to half its flying speed."
+					]
+				}
+			],
+			"legendaryGroup": {
+				"name": "White Dragon",
+				"source": "MM"
+			},
+			"fluff": {
+				"_monsterFluff": {
+					"name": "Chromatic Greatwyrms",
+					"source": "BeGr"
+				},
+				"images": [
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "img/bestiary/MM/White%20Dragon.jpg"
+						}
+					}
+				]
+			},
+			"dragonCastingColor": "W",
```

Comment:
Casting colours were recently migrated to be full strigns rather than abbreviations